### PR TITLE
Bug 1224065 - Migrate FxA to use modern L10n API

### PR DESCRIPTION
**Pull request by zbraniecki**
https://bugzilla.mozilla.org/show_bug.cgi?id=1224065

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=742d47c3a2d8d88b76753e9699989cc70fb02e4b

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=bc1f73e9f9e3f78aec7a0d9c3a3a053653f06552

File: apps/system/fxa/elements/fxa-email.html [hunk 1]
```diff
@@ -3,12 +3,12 @@
     <section class="vertical">
       <div id="fxa-logo" class="fxa-logo"></div>
       <p class="fxa-intro" data-l10n-id="fxa-use-email-to-sign-in">
-        Use your email address to create an account or sign in.
       </p>
       <input id="fxa-email-input" data-l10n-id="fxa-enter-email-input" type="email" placeholder="Enter your email" required="" x-inputmode="verbatim">
 
-      <p id="fxa-notice" class="fine-print">
-        By proceeding, I agree to the <a class="disabled">Terms of Service</a> and <a class="disabled">Privacy Notice</a> of Firefox cloud services.
+      <p id="fxa-notice" class="fine-print" data-l10n-id="fxa-notice2">
```
Comment (ferjm) [inline apps/system/fxa/elements/fxa-email.html]: You didn't change the string so I think you don't need to change the string id: `fxa-notice`

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=345638fc61959e4c53f3a905dedea9b258cd9faf

**Pull request merged**